### ISSUE TITLE
Improve dataroot handling

### ISSUE THREAD

**Issue by markmetcalfe** [totara/totara-docker-dev#108]
In the tdb command, the dataroot can be backed up and restored. However, often when setting up a new site we might want to create a fresh dataroot. This still has to be done independently of docker-dev.

I'm thinking it might be nice to have a ```tdataroot``` command that handles creating the dataroot directory - i.e takes in the config.php variables, and does an action in a similar way to tdb. Once that is done, then we can abstract out the backup/restoring of the dataroot from tdb into tdataroot.

With this working, it wouldn't really be necessary to ship the php containers with predefined dataroots. Instead they could just be created/deleted on-demand.

**Comment by derschatta** [comment]
What about having a tsetup command which can create everything from scratch in an automated way, create dataroot, config.php, database and install the site?

If we have a config template file with variables we could use it to populate the final config.php file with the desired content based on options of the tsetup command

**Comment by markmetcalfe** [comment]
Yeah that would be nice and handy.

**Comment by markmetcalfe** [comment]
Have made the dataroots dynamic as part of #165 - I think this is a good enough improvement to dataroot handling for now
